README.md

```diff
@@ -88,7 +88,7 @@ SELECT sum(clicks), sum(impressions) FROM AdAnalyticsTable
 ```
 
 ## Contributing to Pinot
-Please refer to the [Contribution Guide](https://docs.pinot.apache.org/community/contribution) for 
+Please refer to the [Contribution Guide](https://docs.pinot.apache.org/developers/developers-and-contributors) for 
```

Comment:
Maybe we can link to https://docs.pinot.apache.org/developers/developers-and-contributors/contribution-guidelines directly (which has links to the other pre-requisites)?